Zend/asm/jump_x86_64_sysv_elf_gas.S

```diff
@@ -30,6 +30,7 @@
 .type jump_fcontext,@function
 .align 16
 jump_fcontext:
+    endbr64
```

Comment:
If I'm not mistaken, these assembler files are re-used from the [boost libraries](https://www.boost.org/), and as such it would probably be a good idea to provide these modifications upstream.